includes/template-library/sources/local.php

```diff
@@ -9,6 +9,7 @@
 use Elementor\Modules\LandingPages\Module as Landing_Page_Module;
 use Elementor\Modules\Library\Documents\Library_Document;
 use Elementor\Plugin;
+use ElementorPro\Modules\CustomCode\Module;
```

Comment:
Remove the pro code.
Make it generic for all types.

Reply:
Fixed